plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -209,7 +232,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
                EOS_ASSERT(false, plugin_exception, "");
             }
 #endif
-         }), "Override default WASM runtime")
+         })->default_value(eosio::chain::wasm_interface::vm_type::eos_vm_jit, "eos-vm-jit"), wasm_runtime_opt.c_str()
```

Comment:
For 2.0 the default wasm runtime needs to remain wabt. Actually that's curious, why can't we use  `chain::config::default_wasm_runtime` here to keep the default defined in a single location?

Reply:
used chain::config::default_wasm_runtime.